.azure-pipelines/win32-conda-CI.yml

```diff
@@ -46,13 +46,18 @@ jobs:
       echo Test numpy installation... && python -c "import numpy"
       pip install %COREML_PATH% %ONNX_PATH%
       pip install -r requirements-dev.txt
-      echo Test onnxruntime installation... && python -c "import onnxruntime"
```

Comment:
I noticed there was no onnxruntime installation in the win32 build -- when 0.1.4 is added, one of the logistic regression spark ml tests fails. Without 0.1.4, no tests explicitly fail. I've informed @jeffsaremi regarding the log-reg test, but is the lack of onnxruntime in the windows build expected?

Reply:
Will be addressed in separate PR.